### PR TITLE
Note on collation for direct lake dataset

### PR DESCRIPTION
**Pull request by mlonsk**
Created note for collation
Changed dataset to semantic model
Fixed broken link in powerbi-xmla.md

### CODE REVIEW

File: common/Datasets/direct-lake-dataset.md [hunk 1]
```diff
@@ -15,17 +15,20 @@ applies_to:
...
+Tabular Editor 3 can create and connect to this type of dataset. For a tutorial on this please refer to our blog article: [Direct Lake semantic models: How to use them with Tabular Editor](https://blog.tabulareditor.com/2023/08/23/fabric-direct-lake-dataset)
 
-Tabular Editor 2 can connect to Direct Lake datasets, but does not have any built in functionality to create new tables or direct lake datasets. This needs to be done manually or with a C# script. 
+Tabular Editor 2 can connect to Direct Lake semantic models, but does not have any built in functionality to create new tables or direct lake semantic models. This needs to be done manually or with a C# script. 
 
+> [!NOTE]
+> DirectLake models currently use a collation that is different from regular Power BI import semantic models. This may lead to different results when querying the model, or when referencing object names in DAX code.
+For more information please see this blog post by Kurth Buhler: [Case-sensitive models in Power BI: consequences & considerations](https://data-goblins.com/power-bi/case-specific)
```
Comment (otykier) [inline common/Datasets/direct-lake-dataset.md:31]: Add `>` so that the "For more information..." text is kept inside the NOTE infobox

**Review by otykier — CHANGES_REQUESTED** [review]

File: common/Datasets/direct-lake-dataset.md (hunk 1 above)
Comment (mlonsk) [reply common/Datasets/direct-lake-dataset.md:31]: It is :)
image

Comment (mlonsk) [reply common/Datasets/direct-lake-dataset.md]: Hiding in shame 😳

File: common/Datasets/direct-lake-dataset.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Tabular Editor 2 can connect to Direct Lake semantic models, but does not have a
 
 > [!NOTE]
 > DirectLake models currently use a collation that is different from regular Power BI import semantic models. This may lead to different results when querying the model, or when referencing object names in DAX code.
-For more information please see this blog post by Kurth Buhler: [Case-sensitive models in Power BI: consequences & considerations](https://data-goblins.com/power-bi/case-specific)
+For more information please see this blog post by Kurt Buhler: [Case-sensitive models in Power BI: consequences & considerations](https://data-goblins.com/power-bi/case-specific)
```
Comment (otykier) [inline common/Datasets/direct-lake-dataset.md:31]: I still don't see the '>' in front of this line? :)

Comment (mlonsk) [reply common/Datasets/direct-lake-dataset.md:31]: If you insist I am happy to add it, but it works just fine without

Comment (otykier) [reply common/Datasets/direct-lake-dataset.md]: You're right. That was actually surprising to me as I thought all lines had to be prefixed by that. However, there's a bug in the way the link is shown in the infobox below:

image

Comment (mlonsk) [reply common/Datasets/direct-lake-dataset.md]: @otykier The link has been fixed

**Review by otykier — APPROVED** [review]

**Pull request merged**